src/DwhProvider/DwhLocationEnum.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DbtTransformation\DwhProvider;
+
+enum DwhLocationEnum
```

Comment:
Asi blbost, jen přemýšlím jestli ten naming je dost clear. Protože location už tu někde používáme u regionů, u konkrétních remote DWH. Jestli by nebylo jasnější něco jako `DwhAccessTypeEnum`, `DwhConnectionTypeEnum`, `DwhEnvironmentTypeEnum` nebo zachovat klidně to location a přidat ten type `DwhLocationTypeEnum` 🤷‍♂️

Reply:
ok, upravil jsem na `DwhConnectionTypeEnum`

Reply:
Ale upravil bych to teda i všude jinde. Teď je to mixnutý v těch názvech proměnných atd.

Reply:
Upraveno, jsem si toho nevšimnul po renamu ve stormu